rm_vk/rm_vk/NewsFeedViewController/Models/NewsFeed.swift

```diff
@@ -0,0 +1,15 @@
+// NewsFeed.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Новости
```

Comment:
```suggestion
/// Новости
```
структура другая, а описание тоже